src/foam/util/Emails/EmailsUtility.java

```diff
@@ -52,6 +54,11 @@ public static void sendEmailFromTemplate(X x, User user, EmailMessage emailMessa
         templateArgs = new HashMap<>();
         templateArgs.put("template", templateName);
       }
+      DAO themeDAO = (DAO) x.get("themeDAO");
+      Theme theme = (Theme) themeDAO.find(
+            MLang.EQ(Theme.SPID, user.getSpid())
+          );
+      templateArgs.put("theme", theme);
```

Comment:
This is inappropriate.  we'll add theme to the context (via Session), then this is not necessary.  And having an entire modeled object as an argument is silly.   These should be key/value pairs.